tap_quickbooks/schemas/time_activities.json

```diff
@@ -98,6 +98,57 @@
         "integer"
       ]
     },
+    "BreakHours": {
```

Comment:
As per the [doc](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/timeactivity), BreakHours and BreakMinutes fields are added in a single value as `BreakHours BreakMinutes`. But in the API response, both fields are replicated as different fields.
![image](https://user-images.githubusercontent.com/80324346/159691237-7a0d3875-21d0-4085-9e74-1f185d0849e8.png)
Hence, segregated these fields in the schema.